selfdrive/car/volkswagen/values.py

```diff
@@ -392,18 +392,24 @@ def init_make(self, CP: car.CarParams):
   CAR.CRAFTER_MK2: {
     (Ecu.engine, 0x7e0, None): [
       b'\xf1\x8704L906056EK\xf1\x896391',
+      b'\xf1\x8705L906023BC\xf1\x892688',
     ],
     # Only current upstreamed vehicle has a manual transmission
     #(Ecu.transmission, 0x7e1, None): [
     #],
     (Ecu.srs, 0x715, None): [
       b'\xf1\x873Q0959655BG\xf1\x890703\xf1\x82\x0e16120016130012051G1313052900',
+      b'\xf1\x875QF959655AG\xf1\x890753\xf1\x82\x13151400151500111111000502----1311120749',
+      b'\xf1\x875QF959655AS\xf1\x890755\xf1\x82\x1315140015150011111100050200--1311120749',
     ],
     (Ecu.eps, 0x712, None): [
       b'\xf1\x872N0909143E \xf1\x897021\xf1\x82\x05163AZ306A2',
+      b'\xf1\x872N0909144K \xf1\x897045\xf1\x82\x05233AZ810A2',
     ],
     (Ecu.fwdRadar, 0x757, None): [
       b'\xf1\x872Q0907572M \xf1\x890233',
+      b'\xf1\x872Q0907572R \xf1\x890372',
+      b'\xf1\x872Q0907572AA \xf1\x890396',
```

Comment:
There's an extra space here that's probably a manual transcription error. Also, the AA one should be sorted at the top. It's chronologically wrong but alphabetically right, and what we're doing elsewhere for consistency.

```suggestion
      b'\xf1\x872Q0907572AA\xf1\x890396',
```